hazelcast/src/test/java/com/hazelcast/config/ConfigXmlGeneratorTest.java

```diff
@@ -156,8 +160,13 @@ public void testReplicatedMapConfigGenerator() {
                 .setStatisticsEnabled(false)
                 .setConcurrencyLevel(128)
                 .setQuorumName("quorum")
+                .setInMemoryFormat(InMemoryFormat.NATIVE)
```

Comment:
Could you please revert these `ReplicatedMap` changes? They are already part of https://github.com/hazelcast/hazelcast/pull/12196 as well.